patches/expect-puppeteer+6.1.1.patch

```diff
@@ -1,9 +1,42 @@
+diff --git a/node_modules/expect-puppeteer/lib/index.js b/node_modules/expect-puppeteer/lib/index.js
```

Comment:
As this been patched upstream?

Reply:
Not yet, which is why we need this here for the time being.